side-cars/build/Dockerfile.nsm-init-arm64

```diff
@@ -0,0 +1,17 @@
+FROM golang:alpine as build
+RUN apk --no-cache add git
+ENV PACKAGEPATH=github.com/networkservicemesh/networkservicemesh/
+ENV GO111MODULE=on
+
+RUN mkdir /root/networkservicemesh
+ADD ["go.mod","/root/networkservicemesh"]
+ADD ["./scripts/go-mod-download.sh","/root/networkservicemesh"]
+WORKDIR /root/networkservicemesh/
+RUN ./go-mod-download.sh
+
+ADD [".","/root/networkservicemesh"]
+RUN GOARCH=arm64 CGO_ENABLED=0 GOOS=linux go build -ldflags '-extldflags "-static"' -o /go/bin/nsm-init ./side-cars/cmd/nsm-init/init.go
```

Comment:
Please consider reusing the existing DOcker files and parametrise the GOARCH. See how we do it for VPP version.

Reply:
Ok, I will try.

Reply:
This is partly granted in commit 2c28e76 . I will finish the rest of Dockerfiles in the same style...